logos/benchmarks/benchmark_logos.py

```
@@ -4974,102 +5801,197 @@ def _cleanup(reason: str = "cleanup") -> None:
+            unique_models = list(dict.fromkeys(e.body["model"] for e in workload if e.body.get("model")))
+            excluded = [m for m in unique_models if m in dynamo_skip]
+            unique_models = [m for m in unique_models if m not in dynamo_skip]
+            if excluded:
+                print(f"  [dynamo] Excluding model(s) {excluded} (not served by Dynamo on this cluster).")
+            placements, skipped = _dynamo_plan_placement(
+                unique_models, args.gpu_host, gpus_per_node, _DYNAMO_MODEL_NUM_GPUS
+            )
+            if skipped:
+                print(
+                    f"  [dynamo] WARNING: no GPU capacity for {skipped} "
+                    f"({len(args.gpu_host)} host(s) × {gpus_per_node} GPU) — those models will 404.",
```

Ruff (0.15.18)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)